src/schema/v2/user.ts

```diff
@@ -134,7 +138,36 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
     dataTransferOptOut: {
       description: "Has the user opted out of data transfer.",
       type: GraphQLBoolean,
-      resolve: ({ data_transfer_opt_out }) => data_transfer_opt_out,
+      resolve: async ({ data_transfer_opt_out }) => data_transfer_opt_out,
+    },
+    interestsConnection: {
+      type: UserInterestConnection,
+      args: pageable({}),
+      resolve: async ({ id }, args, { userInterestsLoader }) => {
+        if (!userInterestsLoader) {
+          throw new Error(
+            "Loader not found. You must supply an X-Access-Token header."
+          )
+        }
+
+        const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+        const { body, headers } = await userInterestsLoader(id, {
+          page,
+          size,
+          total_count: true,
+        })
+
+        const totalCount = parseInt(headers["x-total-count"] || "0", 10)
+
+        return {
+          totalCount,
+          ...connectionFromArraySlice(body, args, {
+            arrayLength: totalCount,
+            sliceStart: offset,
+            resolveNode: (node) => node.interest,
```

Comment:
This is important, it tells the helper to take whatever is in the `interest` property of the returned data (which is the full artist or gene), and make that the `node` field:

![Screen Shot 2022-08-29 at 4 37 57 PM](https://user-images.githubusercontent.com/1457859/187294305-2bc2533b-863b-4f98-8a6a-b6d69099bb87.png)